app/services/live.js

```diff
@@ -297,6 +300,20 @@ export default class LiveService extends Service {
 
   async renderScreenVideoToPeers(peers) {
     this.peers = peers;
+    const rolesToFind = [ROLES.maven, ROLES.moderator, ROLES.guest];
+    const foundPeers = {};
+
+    for (const roleName of rolesToFind) {
+      foundPeers[roleName] = peers.find((peer) => peer.roleName === roleName);
+    }
```

Comment:
still we are running loop for each roleName, and shouldn't `foundPeers[roleName] ` be a list ?

Reply:
no, because we need to track the peer's array to if there is any role name with maven, moderator, or guest present in that, and if not then it should show a message.... I did try some options  but nothing worked